libs/build_esma_cmake.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eux
+
+name="esma_cmake"
+repo=${1:-${STACK_esma_cmake_repo:-"GEOS-ESM"}}
```

Comment:
Are there multiple repos for esma_cmake?
If not, we should only  install the official one.
Please remove `$repo` from the prefix path and modulefiles.

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884